tests/test_tomography.py

```diff
@@ -69,7 +69,7 @@ def test_fidelity_cnot_operator():
     cnot_p = catalog["klm cnot"].build_processor()
     cnot_op = np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]], dtype='complex_')
 
-    cnot_fidelity = fidelity_op_process_tomography(cnot_op, cnot_p, nqubit=2)
+    cnot_fidelity = fidelity_op_process_tomography(cnot_op, cnot_p)
 
     assert cnot_fidelity == pytest.approx(1, 1e-3)  # computed fidelity is around 0.99967
```

Comment:
If you rebase / merge from release/0.10.0 you will have a better fidelity since I fixed the KLM CNOT coefficients

Reply:
Could you do this while merging the PR?